cleanlab_studio/internal/api/api_helper.py

```diff
@@ -9,3 +13,49 @@ def check_uuid_well_formed(uuid_string: str, id_name: str) -> None:
         raise InvalidUUIDError(
             f"{uuid_string} is not a well-formed {id_name}, please double check and try again."
         )
+
+
+def extract_df_subset(
+    df: pd.DataFrame, subset_indices: Union[Tuple[int, int], List[int], None]
+) -> pd.DataFrame:
+    """Extract a subset of the dataframe based on the provided indices. If no indices are provided, the entire dataframe is returned. Indices can be range or specific row indices."""
+    if subset_indices is None:
+        return df
```

Comment:
```suggestion
    if subset_indices is None:
        print("Running on your full dataset since `subset_indices` is None. This may take a while. Specify this argument to get faster results for a subset of your dataset.")
        return df
```